### DISCUSSION TITLE
v2.223

### DISCUSSION THREAD

**Discussion by anddea** [anddea/revanced-patches#241] [Announcements]
I've seen a few questions:
1. If you have performance issues, it means that you are hiding or adding a lot of elements. It depends on the performance of your device and the number of patches/options enabled. Disable debug logging patch which should be disabled by default.
2. If there are no elements in the interface, such as buttons, shelves, etc., you can go to the settings and turn them back on, this is not a bug.

Vote in these [polls](https://github.com/anddea/revanced-patches/discussions/categories/polls). I welcome other developers and designers who'd like to help us build this project.

**Comment by 54m4d** [comment]


**Reply by anddea** [reply]
> settings appear to be missing

It's a patch `Tuck away preferences`.

**Reply by anddea** [reply]
This patch's preference isn't available inside YouTube settings. In manager, click on gear icon and put the exact title of preferences in main settings screen in English only (it will remove for all languages but in Manager you have to type in English), separated by comma. You can use Revancify or anything you want to set it up.

**Comment by fcaronte** [comment]
Buffer Fix is needed on microg part? Actually I have a lot of buffering problems

**Reply by anddea** [reply]
What buffering problems do you have? Latest patches will redirect you to download the latest GmsCore by ReVanced. It'll fix content not available.

**Reply by fcaronte** [reply]
Yes i mean the stuck during playing video after few seconds "like you miss connection" I think is called buffer fix but should be done on microg side as I understand

**Reply by anddea** [reply]
If you face this same buffer error in stock app, then there's no solution to that. It's Google doing something weird. 

Otherwise, try to to turn on and off incognito mode. Or turn on and off spoof app version.

**Reply by anddea** [reply]
Or create a new account. It can fix the first problem, if you have buffer issues in stock app too.

**Reply by fcaronte** [reply]
Thanks for suggestion, stock app Is okay and smart tube on android TV Is working correctly too

**Reply by anddea** [reply]
Try to go to GmsCore and change profile (maybe set emulator arm64 if you have arm64) in device registration setting, maybe it will help you.

**Comment by Dhyfer1** [comment]
@anddea What version of gmscore do you recommend using with your patches? The latest gmscore version of inotia00, Revanced, YT-Advanced or MicroG-RE?

**Reply by vippium** [reply]
Gmscore by inotiaa....wen?

**Reply by vippium** [reply]
U can give a try to rufusin gmsvore...he added "Hide to launcher" recently

**Reply by Dhyfer1** [reply]
This https://github.com/inotia00/VancedMicroG

**Reply by anddea** [reply]
You can use whatever suits the best, they are all fine, inotia00's is not compatible, as it uses old MicroG.

**Reply by Dhyfer1** [reply]
Ok anddea, thanks for reply 👍

**Comment by NikunjK8** [comment]
Should i use Revanced manager or rvx manager? Also is it true that i need stock youtube app installed for patching to work? (Currently have it removed through adb). If yes then do i need to update yt to latest version as well or my stock OEM version will work?

**Reply by anddea** [reply]
Use whatever you like, RVX Manager shows dev releases that can be broken sometimes because they're in a development. ReVanced Manager shows only stable. 

You don't have to install stock app.

**Comment by n4zk4** [comment]
I can't seem to find a recommended YouTube version for your patches. Is there any or can I just patch the most recent one? If there's a recommended version, where can I see it?

**Reply by NikunjK8** [reply]
Its 19.11.38
https://github.com/anddea/revanced-patches?tab=readme-ov-file#-json-format

**Reply by n4zk4** [reply]
Cool, thank you! I actually had seen that and was using 19.11.38, but I wasn't sure as it said "Example" and thought maybe it was outdated.

**Comment by Gevorg102** [comment]
There is no option to download the video, it says com.deniscerri.ytdl is unavailable

**Reply by anddea** [reply]
You have to download any downloader and put its package name there, for default you have to install [ytdlnis](https://github.com/deniscerri/ytdlnis/releases)

**Comment by 54m4d** [comment]
Hello Anddea
Do you have any idea hot to get rid of this button? It's called "Play Something"

<details>
<summary>Screenshot</summary>
<img src=https://github.com/anddea/revanced-patches/assets/52612348/10ff6d11-d818-45ad-8c4f-ed4be471315f>
</details>

**Reply by anddea** [reply]
Hi. I'll take a look. Although I cannot see this button in my accounts, not sure when I'll be able.

**Reply by 54m4d** [reply]
Hello, I'm bothered by the fact that this button is constantly visible on the homepage, so I've begun turning on different patches to see if any improvements can be made. I've discovered that toggling on `'Hide floating microphone'` causes the button in question to disappear.
Perhaps now this can provide you with an understanding of what this could be? Good luck!

**Reply by anddea** [reply]
So `Hide floating microphone` hides this button as well? If so, then maybe it's okay to combine it and name `Hide play something and floating microphone`. I still don't see this button, so I can't verify it.

**Reply by 54m4d** [reply]
Or just `'Hide play something button'` since the floating microphone does not exist anymore, at least for me, maybe it's deprecated? However, I'm not sure if this patch has any negative side effects, though, but for now it looks good.

You can try to commit this name change (dev), and I'll test it, if there are any issues I'll provide feedback.

**Comment by 54m4d** [comment]
Are there patches to hide these kinds of warnings and buttons in live chat?

<details>
<summary>Screenshot</summary>
<img src=https://github.com/anddea/revanced-patches/assets/52612348/fa43a008-9bd2-41f0-8398-9f2300679aba>
</details>

**Reply by anddea** [reply]
No, nothing like that from what I remember.

**Reply by 54m4d** [reply]
Thanks, I wonder if it's possible to create patches to address these elements. I'm uncertain why they haven't been made available yet, maybe there must have been a reason?

**Reply by anddea** [reply]
I couldn't find related filter yet, so not possible for now.

**Reply by 54m4d** [reply]
Makes sense now, Thanks.

**Comment by 54m4d** [comment]
This is bad news.

https://support.google.com/youtube/thread/269521462?hl=en

**Reply by anddea** [reply]
I've seen news about it. 

What hypocrites.
> We want to emphasize that our terms don’t allow third-party apps to turn off ads because that prevents the creator from being rewarded for viewership, and Ads on YouTube help support creators and let billions of people around the world use the streaming service. 

It's YouTube itself that

> prevents the creator from being rewarded for viewership

And ads on YouTube don't

> help support creators 

They just make rich YouTubers more rich and their Terms Of Service kill creators harder than we do. If YouTube paid enough, creators wouldn't add sponsors to their videos. And if you really want to support channels, just add channel to whitelist of SponsorBlock or disable it at all. I guess we wouldn't need SponsorBlock if YouTube recommendations worked fine. 

But instead of fixing bugs to improve UX they try to kill third party apps just because they are so greedy that they want even more money (not for creators obviously, but for themselves).
